src/Controller/Ast/ProcessAstFormController.php

```diff
@@ -6,25 +6,20 @@
 
 use Illuminate\Http\RedirectResponse;
 use Illuminate\Routing\Controller;
-use Rector\Website\Entity\AstRun;
-use Rector\Website\Repository\AstRunRepository;
+use Rector\Website\Ast\Entity\AstRun;
 use Rector\Website\Request\AstFormRequest;
-use Symfony\Component\Uid\Uuid;
 
 final class ProcessAstFormController extends Controller
 {
-    public function __construct(
-        private AstRunRepository $astRunRepository
-    ) {
-    }
-
     public function __invoke(AstFormRequest $astFormRequest): RedirectResponse
     {
-        $astRun = new AstRun(Uuid::v4(), $astFormRequest->getPhpContents());
-        $this->astRunRepository->save($astRun);
+        $astRun = new AstRun();
+        $astRun->content = $astFormRequest->getPhpContents();
+        $astRun->hash = sha1($astFormRequest->getPhpContents());
```

Comment:
The route validation is using uuid, so it should use uuid as well

https://github.com/rectorphp/getrector-com/blob/c6e5a95fce227082f6f345d05de941c55853073a/routes/web.php#L60

Reply:
Updated to "hash" :+1: